src/lightning_habana/fabric/plugins/precision.py

```diff
@@ -55,6 +57,11 @@ def __init__(
         fp32_file_path: Optional[str] = None,
         verbose: bool = False,
     ) -> None:
+        rank_zero_info(
+            "The 'HPUPrecision' strategy is deprecated and will be removed in lightning-habana release 1.12.0,"
```

Comment:
Do you mean Habana drivers version here as `lightning-habana` is now 1.0 so to 1.12 is long path

Reply:
Sorry. It must be 1.2.0. Will correct it.